packages/broker/src/plugins/operator/MaintainTopologyHelper.ts

```diff
@@ -17,23 +17,23 @@ export interface OperatorClientEvents {
     /**
      * Emitted when staking into a Sponsorship on a stream that we haven't staked on before (in another Sponsorship)
      */
-    addStakedStream: (streamId: string, blockNumber: number) => void
+    addStakedStream: (streamIds: string[]) => void
 
     /**
      * Emitted when a unstaked from ALL Sponsorships for the given stream
      */
-    removeStakedStream: (streamId: string, blockNumber: number) => void
+    removeStakedStream: (streamId: string) => void
 }
 
-export interface OperatorClientConfig {
+export interface TopologyHelperConfig {
```

Comment:
For consistency I would name:
```suggestion
export interface MaintainTopologyHelperConfig {
```